src/vaadin-select.html

```diff
@@ -443,21 +461,23 @@
           }
         }
 
-        /** @protected */
+        /**
+         * @return {!HTMLElement}
+         * @protected
+         */
         get focusElement() {
           return this._inputElement ||
             (this._inputElement = this.shadowRoot.querySelector('vaadin-select-text-field'));
         }
 
-        /** @private */
+        /** @protected */
         disconnectedCallback() {
           super.disconnectedCallback();
           this.removeEventListener('iron-resize', this._boundSetPosition);
           // Making sure the select is closed and removed from DOM after detaching the select.
           this.opened = false;
         }
 
-        /** @private */
```

Comment:
We have same problem as in `vaadin-split-layout`: no way to annotate `IronResizableBehavior`. Previously we agreed that we will mark `notifyResize` as public because of that. So I did the same change here.